### PR TITLE
CFINSPEC-82: Add ppa resource 

### PR DESCRIPTION
**Pull request by ahasunos**
Signed-off-by: Sonu Saha [sonu.saha@progress.com](mailto:sonu.saha@progress.com)
## Description

**ISSUE: CFINSPEC-82 — Add ppa resource.**
`ppa` resource allows to verify PPA repositories on Debian-based linux distributions
## Related Issue

This resource would allow to verify PPA respositories using the `ppa` resource. Existing verification of PPA repositories by `apt` resource remains as it is. User can use either `ppa` or `apt` resource to verify PPA repositories on Debian-based linux distributions.
## Changes made to existing file:
- etc/deprecations.json
- lib/inspec/resources/apt.rb

## New files included for lxc resource:
- docs-chef-io/content/inspec/resources/ppa.md
- test/unit/resources/ppa_test.rb

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by IanMadd** [comment]
The Netlify build is failing because of a change that I made in chef-web-docs. I updated the build script here in #5927. Rebase this PR and Netlify should work fine. Let me know if it doesn't work.

**Comment by ahasunos** [comment]
> The Netlify build is failing because of a change that I made in chef-web-docs. I updated the build script here in #5927. Rebase this PR and Netlify should work fine. Let me know if it doesn't work.

It works @IanMadd. Thanks

File: docs-chef-io/content/inspec/resources/ppa.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `ppa` Chef InSpec audit resource verifies PPA repositories on Debian-based linux distributions
+
+    describe ppa('path) do
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/resources/ppa.md]: Missing closing quote

Comment (ahasunos) [reply docs-chef-io/content/inspec/resources/ppa.md]: Fixed!

**Review by clintoncwolfe — APPROVED** [review]
Looks good to me!

**Review by dkumaras — APPROVED** [review]

**Review by ahasunos — COMMENTED** [review]
@dkumaras I think the DCO test is failing because of the missing signed-off.

**Pull request merged**